ccube.nim

```diff
@@ -14,23 +14,23 @@ Distributed under GPLV3
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 ]#
-import std/[os,strutils,strformat,random,json]
+import std/[os, strutils, strformat, random, json, parseopt]
 import bigints
 
 randomize()
 
-let VER = "1.0.0"
-let banner = fmt"""
+const VER = "1.0.0"
+const banner = fmt"""
 ░█████╗░░█████╗░██╗░░░██╗██████╗░███████╗
 ██╔══██╗██╔══██╗██║░░░██║██╔══██╗██╔════╝
 ██║░░╚═╝██║░░╚═╝██║░░░██║██████╦╝█████╗░░
 ██║░░██╗██║░░██╗██║░░░██║██╔══██╗██╔══╝░░
 ╚█████╔╝╚█████╔╝╚██████╔╝██████╦╝███████╗
 ░╚════╝░░╚════╝░░╚═════╝░╚═════╝░╚══════╝Version:{VER}"""
 
-proc showUsage():void =  
-  echo(banner)
-  echo("""
+proc showUsage =
+  echo banner
```

Comment:
Generally in Nim people don't use parenthesis when calling `echo` like that, so I went ahead and changed it